docs/sphinx/index.rst

```diff
@@ -90,16 +90,7 @@ Module & Command Development
     :glob:
     :maxdepth: 1
     :caption: Architecture and Modules
-
+    
     development/*
 
-Contribution and Style Guides
```

Comment:
Seems like this was removed by mistake. I believe the only change needed are the numbers